megatron/neox_arguments/neox_args.py

```diff
@@ -105,6 +105,10 @@ class NeoXArgsModel(NeoXArgsTemplate):
     Transformer hidden size.
     """
 
+    intermediate_size: int = None
```

Comment:
We should include in the help string here that this is (in general) discouraged and that the code will pick something nice for you if you omit it.